tests/unit/auth/v2/queryAuthCheck.js

```diff
@@ -31,10 +31,15 @@ describe('v2: queryAuthCheck', () => {
 });
 
 describe('v2: queryAuthCheck', () => {
-    let clock = sinon.useFakeTimers();
-
-    beforeEach(() => process.env.PRE_SIGN_URL_EXPIRY = 604800000);
+    let clock;
 
+    beforeEach(() => {
+        clock = sinon.useFakeTimers();
+    });
+    afterEach(() => {
+        process.env.PRE_SIGN_URL_EXPIRY = 604800000;
```

Comment:
Shouldn't we set this in the beforeEach so it takes affect in each test?

Reply:
Earlier I put it in beforeEach thinking like that, but essentially the goal was testing it for `undefined` cases as well, so put it inside afterEach so that the first two test cases can test against `undefined` env var.